File: videojuegoDeChinpokomon/src/Ataque/CañonConico.java
package Ataque;

import Chimpokomon.Chimpokomon;

public class CañonConico extends Ataque {

    public CañonConico(Integer dañoValor, String nombre) {
        super(dañoValor, nombre);

    }

    @Override
    public void dañar(Chimpokomon chipoAtacante, Chimpokomon chimpoEnemigo) {
        if (chipoAtacante.tieneVentajaSobre(chimpoEnemigo)) {
            chimpoEnemigo.restarVida(5);
        } else {
            chimpoEnemigo.restarVida(4);
        }

    }

}

File: ChimpokomonPython/ModuloSRC/Ataque/CañonConico.py
from Ataque.Ataque import Ataque

class CañonConico(Ataque):
    def __init__(self):
            Ataque.__init__(self, 2, "CañonConico")

    def dañar(self, chinpokomonAtacante, chimpokomonEnemigo):
        if(chinpokomonAtacante.tieneVentajaSobre(chimpokomonEnemigo)):
            chinpokomonAtacante.restarVida(5)
        else:
            chimpokomonEnemigo.restarVida(4)
